src/main/java/org/template/recommendation/URModel.java

```diff
@@ -16,6 +17,7 @@
 import java.util.*;
 
 /** Universal Recommender models to save in ES */
+@AllArgsConstructor
 public class URModel {
 
     private transient static final Logger logger = LoggerFactory.getLogger(URModel.class);
```

Comment:
https://projectlombok.org/features/Constructor.html

Based on this I feel like the logger might be included as part of the constructor..? O: